README.md

```diff
@@ -76,50 +84,45 @@ To use Device Simulator Express, install the extension from the marketplace and
 
 #### 1. Start with the "Device Simulator Express [Circuit Playground Express]: New File" Command.
 
-1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL+SHIFT+P` to open the command palette).  
-   <img alt='"New File" animation' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/newFile.gif>
-2. Name and save your file somewhere, and we’re good to go! (3)
+1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL/CMD + SHIFT + P` to open the command palette).  

```

Comment:
Here the Windows/Mac shortcuts are formatted as: `CTRL/CMD + SHIFT + P`
Below it's formatted as: `Ctrl + Shift + P` / `Cmd + Shift + P`
Perhaps format all the shortcuts consistently?
I think the second approach is more clear for beginners, since it explicitly mentions which shortcut is for which OS. Up to you!
Also keys like `Ctrl` and `Shift` are sometimes capitalized and sometimes all caps.

Reply:
Changed :D